Policies/Kubernetes/container-no-privilege-escalation/template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazurecontainernoprivilegeescalation
```

Comment:
Don't we need to specify listKind, singular, plural here? Please check with Sertac or Rita.